packages/cli/src/Generate.ts

```diff
@@ -234,8 +234,21 @@ ${chalk.dim('```')}
 ${highlightTS(`\
 import { PrismaClient } from '${importPath}'
 const prisma = new PrismaClient()`)}
-${chalk.dim('```')}${breakingChangesStr}${versionsWarning}`
+${chalk.dim('```')}${
+          prismaClientJSGenerator.options?.dataProxy
+            ? `
+
+To use Prisma Client in edge environments like Cloudflare Workers or Vercel Edge Functions, import it like this:
```

Comment:
```suggestion
To use Prisma Client in Edge runtimes like Cloudflare Workers or Vercel Edge Functions, import it like this:
```
Maybe?

Reply:
I don't think "Edge" should be capitalized here, it's not a proper name or brand, "edge computing" is a generic term. Same as we don't capitalize, e.g., "cloud". No preference on "environments" vs "runtimes", but our docs use "runtimes" too, so I'm gonna change it for consistency as well 👍